app/views/projects/show.html.erb

```diff
@@ -269,175 +180,289 @@
     </div>
   <% end %>
 
-  <div class="mt-6 bg-white shadow overflow-hidden sm:rounded-lg">
-    <div class="px-4 py-5 sm:p-0">
-      <dl>
-        <% if @project.short_description.present?  %>
-          <div class="sm:grid sm:grid-cols-3 sm:gap-4 sm:px-6 sm:pt-5 sm:pb-2">
-            <dt class="text-sm leading-5 font-medium text-gray-500">
-              Short description
-            </dt>
-            <dd class="mt-1 text-sm leading-5 text-gray-900 sm:mt-0 sm:col-span-2">
-              <%= simple_format Rinku.auto_link(sanitize(@project.short_description), :all, 'class="text-indigo-500" rel="nofollow noopener noreferrer"'), { class: 'mb-3' }, sanitize: false %>
-            </dd>
+  <div class="mt-6 mx-4 gap-0 rounded-lg bg-white overflow-hidden sm:mx-0 sm:grid sm:grid-cols-2">
```

Comment:
I find the panels to have a very low contrast against the background. I know it's not in the design but I would add a `shadow-md` class to accentuate that contrast.